powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/fido2/PowerAuthAuthenticatorProvider.java

```diff
@@ -265,7 +265,8 @@ private AuthenticatorDetail convert(Activation activation, ApplicationEntity app
         try {
             authenticatorDetail.setExtras(objectMapper.readValue(activation.getExtras(), new TypeReference<HashMap<String,Object>>() {}));
         } catch (JsonProcessingException e) {
-            //
+            logger.warn(e.getMessage(), e);
```

Comment:
Indeed, improved, better than swallowing the exception but may we use `Optional`?

Reply:
Sure, we can use `Optional`.